_data/speakers.yml

```diff
@@ -0,0 +1,215 @@
+
+    His particular interests include PHP integration with office suites, Textual analysis, Geodata and geographic information systems, and "big data" and data analysis.
+  talks:
+    - title: A Functional Guide to Cat Herding with PHP Generators
+      abstract: |
+        When working with arrays in PHP, three of the most useful functions available to us are arraymap(), arrayfilter() and array_reduce(), which allow us to walk an array and manipulate the value of array elements, select a subset of values from an array, or reduce an array to a single value; all using a callback function to determine exactly what logic should be applied. The use of the callback makes them extremely flexible, and these functions can be particularly powerful, especially when combined (or chained) together.
+         However, these functions only work with standard PHP arrays; so if we are using Generators as a data source instead of an array, then we can't take advantage of the functionality that they provide. Fortunately, it's very easy to emulate that functionality and apply it to Generators (and also to other Traversable objects like SPL Iterators), giving us access to all of the flexibility and power that mapping, filtering and reducing can offer, and with all the benefits that Generators can offer our code.
+
+        So how do we go about implementing filter(), map() and reduce() functions for our Generators? How do we use those functions in our applications? I'll be answering those two questions, and explaining how I keep track of my cats armed only with a GPS Tracker and, PHP Generators, and filter/map/reduce.
+
+davies:
+  name: Oliver Davies
+  avatar: davies.jpg
+  twitter: opdavies
+  bio: |
+    Oliver Davies has been building websites since 2007, and specialising in PHP and Drupal since 2008. He is currently a Lead Developer at [CTI Digital](https://www.ctidigital.com), is a Drupal 7 and 8 core contributor, and a provisional member of the Drupal Security team. In Oliver's spare time, he enjoys spending time with his family, co-organises the [Drupal Bristol](http://www.drupalbristol.org.uk) and [PHPSW](https://phpsw.uk) user groups, and is a [DrupalCamp Bristol](http://www.drupalcampbristol.co.uk) committee member. You can follow him on Twitter at [@opdavies](https://twitter.com/opdavies).
+  talks:
+    - title: Drupal 8 - Rejoining the Herd
+      abstract: |
+        [Drupal 8](https://www.drupal.org/8) was (finally) released on November 19th 2015, after almost 4 years of work and code commits by over 3,200 different contributors. Whilst it's pretty much the same as the Drupal that we know and, hopefully, love, a lot has changed behind the scenes and under the hood!
```

Comment:
So if anyone wants to go through these and make them HTML links that would be good :)